controls/accordion/states.md

```diff
@@ -12,7 +12,7 @@ The purpose of this help article is to show you the expanded/collapsed states of
 
 ## Collapsed/Expanded States
 
-In the Accordion control each item provides a header that expands when clicked, showing more information. The control is designed in such a way that opening one AccordionItem automatically closes the previously displayed content. 
+In the Accordion control each item provides a header that expands when clicked, showing more information. The control is designed in such a way that opening one `AccordionItem` automatically closes the previously displayed content. 
```

Comment:
Missing comma after the prepositional phrase at the beginning of the sentence.
A prepositional phrase is a group of words that begins with a preposition and includes a noun or pronoun, which is the object of the preposition. For example, "In the park, the trees had lost their leaves."